README.md

```diff
@@ -77,6 +80,8 @@ A list of UI components built with Material-UI Design.
 
 A list of mature apps built with Material-UI Design.
 
+- [Material-ui Medium Blog](https://github.com/sabesansathananthan/material-ui-medium-blog)
```

Comment:
Should be changed ad Material-**UI**
and please add a description after '-' sign

eg: bethesda.net - The official site for Bethesda, publisher of Fallout, DOOM, Dishonored, Skyrim, Wolfenstein, The Elder Scrolls, more.